accname/name/comp_host_language_label.tentative.html

```diff
@@ -0,0 +1,86 @@
+    <input id="lirp" type="text" data-expectedlabel="input label" data-testname="html: label[for][role none] input[role none]" class="ex-label" role="none">
+    <h3>HTML label encapsulation</h3>
+    <label role="none">
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: input[role none] with label[role none] encapsulation" class="ex-label" role="none">
+    </label>
+
+    <h2>HTML input with role="none" and disabled with label associated</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irpd">input label</label>
+    <input id="irpd" type="text" data-expectedlabel="" data-testname="html: label[for]input[role none][disabled]" class="ex-label" role="none" disabled>
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="" data-testname="html: input[role none][disabled] with label encapsulation" class="ex-label" role="none" disabled>
+    </label>
+
+    <h2>HTML img with role="none" and non empty alt</h2>
+    <img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" role="none" alt="w3c logo" data-expectedlabel="" data-testname="html: img[role none][non-empty alt]" class="ex-label">
+
+    <h2>SVG wrapping title with role="none"</h2>
+    <svg viewBox="0 0 20 10" xmlns="http://www.w3.org/2000/svg" data-expectedlabel="circle" data-testname="svg: svg wrapping title[role none]" class="ex-label">
+      <circle cx="5" cy="5" r="4">
+        <title role="none">circle</title>
+      </circle>
+    </svg>
+
+    <h2>HTML fieldset wrapping legend with role="none"</h2>
+    <fieldset data-expectedlabel="legend" data-testname="html: fieldset wrapping legend[role none]" class="ex-label"><legend role="none">legend</legend></fieldset>
+
+    <h2>HTML fieldset with role="none" wrapping legend</h2>
+    <fieldset role="none" data-expectedlabel="" data-testname="html: fieldset[role none] wrapping legend" class="ex-label"><legend>legend</legend></fieldset>
+
+    <h2>HTML fieldset role="none" wrapping legend with role="none"</h2>
+    <fieldset role="none" data-expectedlabel="" data-testname="html: fieldset[role none] wrapping legend[role none]" class="ex-label"><legend role="none">legend</legend></fieldset>
+
+    <h2>HTML table wrapping caption with role="none"</h2>
+    <table data-expectedlabel="caption" data-testname="html: table wrapping caption[role none]" class="ex-label"><caption role="none">caption</caption></table>
+
+    <h2>HTML table with role="none" wrapping caption</h2>
+    <table role="none" data-expectedlabel="" data-testname="html: table[role none] wrapping caption" class="ex-label"><caption>caption</caption></table>
+
+    <h2>HTML table role="none" wrapping caption with role="none"</h2>
+    <table role="none" data-expectedlabel="" data-testname="html: table[role none] wrapping caption[role none]" class="ex-label"><caption role="none">caption</caption></table>
```

Comment:
Like the fieldset[role=none] test above, this may not be a valid test assumption due to an artifact of the WebDriver methods not defining the expected return type when a DOM element no longer has a backing AX element due to the none role. Wait until all the CI bots complete, since there are implementation detail differences this will likely illustrate.